src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -353,6 +358,25 @@ private void replaceSearchResult(List<MapBean> maps, Pane pane) {
       for (MapCardController mapCardController : mapCardControllers) {
         pane.getChildren().add(mapCardController.getRoot());
       }
+      switch (pane.getId()) {
+        case "recommendedPane":
+          pane.getChildren().add(moreRecommendedButton);
+          break;
+        case "mostLikedPane":
+          pane.getChildren().add(moreMostLikedButton);
+          break;
+        case "newestPane":
+          pane.getChildren().add(moreNewestButton);
+          break;
+        case "mostPlayedPane":
+          pane.getChildren().add(moreMostPlayedButton);
+          break;
+        case "ladderPane":
+          pane.getChildren().add(moreLadderButton);
+          break;
+        default:
+          // Do nothing
+      }
```

Comment:
What's that for UI is supposed to be edited in the fxml

Reply:
This adds the more buttons into the flowpane right after the flowpane has been populated with maps. I don't know if it is possible to do this in fxml while making sure that the button always is the last element.